internal/parser/cedar_marshal.go

```diff
@@ -364,6 +380,8 @@ func astNodeToMarshalNode(astNode ast.IsNode) IsNode {
 		return NodeTypeNot{v, UnaryNode{}}
 	case ast.NodeTypeAccess:
 		return NodeTypeAccess{v}
+	case ast.NodeTypeGetTag:
+		return NodeTypeGetTag{v}
```

Comment:
Did the two cases get separated by accident? It doesn't seem to be alphabetical or some other order I can discern.

Reply:
I've separated them to each respectively be right after the equivalent attribute operation.